scripts/deploy.coffee

```diff
@@ -0,0 +1,24 @@
+# Description:
+#   Causes heart attacks.
+#
+# Commands:
+#   hubot deploy to production - Deploy Ralph to production.
+
+apps = [
+  'argos',
+  'levelup-order-ahead-production',
+  'levelup-production',
+  'order-ahead-network-production',
+  'plutus-production'
+]
+
+emojis = [
+  ':+1:',
+  ':dance:',
+  ':shush:'
+]
+
+module.exports = (robot) ->
+  robot.respond /deploy\s+to\s+prod(?:uction)?/i, (msg) ->
```

Comment:
add word boundary `\b` to first and last words?

Reply:
Ah, good thought, but the pattern is effectively bounded by the bot's name: https://github.com/hubotio/hubot/blob/a5178f5560251edcace24ca73956dbfcb9b3307f/src/robot.coffee#L109

So it'll only ever respond to "ralph deploy…", not "ralph plzdeploy".